docker-app/qfieldcloud/core/management/commands/dequeue.py

```diff
@@ -52,22 +51,16 @@ def handle(self, *args, **options):
 
                 busy_projects_ids_qs = (
                     Job.objects.filter(
-                        status=Job.Status.PENDING,
+                        status__in=[
+                            Job.Status.QUEUED,
+                            Job.Status.STARTED,
+                        ]
                     )
-                    .annotate(
-                        active_jobs_count=Count(
-                            "project__jobs",
-                            filter=Q(
-                                project__jobs__status__in=[
-                                    Job.Status.QUEUED,
-                                    Job.Status.STARTED,
-                                ]
-                            ),
-                        )
-                    )
-                    .filter(active_jobs_count__gt=0)
                     .values("project_id")
+                    .all()
                 )
```

Comment:
But getting them cached means the query gets executed, no? In our case the query from the `busy_project_ids_qs` was never executed in the first place. IMO the `all()` is not needed here, as we will pass this as a subquery later. Or I am missing something?

BTW, your query is much simpler without the annotation, I like it!

Reply:
I think the query cood be cashed between the wait loops and if the jobs updated by worker or app it could get misser here ... But would have to check again ...

Reply:
Ok, checked it. You are right. No effect in the `all()`. So what remains is the simpler query we could keep.